_pages/student-showcase.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: Student Showcase
+permalink: /speak/showcase
+snake: rainbow
+sponsors: true
+---
+
+# Call for Proposals - PyconAU Student Showcase 2019
+After the success of last year’s inaugural Students Showcase, the PyConAU ‘Education Track’ will have a dedicated session for student presentations. 
+
+We are calling for proposals from Australian high school students who are using Python in a project, class assignment or who are hacking on something cool. All levels of experience are welcome from complete novices to Python pros. 
+
+The PyCon AU Conference Education Track is a pre-conference stream that focuses on the use of Python for teaching beginner programmers at high school. The student showcase is an opportunity for your students to exhibit their projects and tell the story of how they were made, and the problems they had to solve.
+
+From the main PyCon AU website: 
+
+“We think that speaking at PyCon AU is a great experience, and prioritise having a broad cross-section of both presenters and topics. Our speakers come from a diverse range of ages, backgrounds, technical knowledge and experience. The topics covered are similarly broad, including different technical areas, topics of general community interest, education, introductory material and specialised areas such as data science and machine learning.” 
+
+We are accepting 10 minute talks from students from year 7 to 12. 
+
+## Important Dates
+- Call for Proposals opens: **Tuesday, May 1 2019**
+- Proposal submission deadline: **Sunday, June 9 2019**
+- Notification of whether proposals have been accepted, not accepted, or waitlisted: **Friday, June 14, 2019**
+- Students whose applications are accepted are invited to (but not required to) attend the entire conference from Friday - Sunday
+
+## Speaker Benefits
+Accepted students will have the opportunity to present their work in front of interested and supportive industry professionals and teachers from other schools. They will be provided with free registration for the specialist events day and both days of the conference main track. Should a speaker also wish to attend the sprints and workshops these will attract a small charge.
+
+If you are able to pay for a full registration, then please consider doing so. The funds from registrations contribute not only to conference costs, but also the financial aid program. Every extra dollar makes it easier for someone else to attend.
+
+## Financial Assistance
+PyCon Australia offers a generous financial assistance program, so that some attendees and speakers have some (or in rare cases, all) of their expenses, such as accommodation or flights, reimbursed to them from the conference budget. PyCon Australia strongly encourages people to apply for financial assistance, you can do that here: https://2019.pycon-au.org/assistance/. We don't want to lose a good talk due to financial need.
```

Comment:
```suggestion
PyCon Australia offers a generous financial assistance program, so that some attendees and speakers have some (or in rare cases, all) of their expenses, such as accommodation or flights, reimbursed to them from the conference budget. PyCon Australia strongly encourages people to apply for financial assistance, take a look at the application process on the [financial assistance page](/assistance/). We don't want to lose a good talk due to financial need.
```